databases.go

```diff
@@ -139,6 +140,7 @@ type DatabasesService interface {
 	UpdatePostgreSQLConfig(context.Context, string, *PostgreSQLConfig) (*Response, error)
 	UpdateRedisConfig(context.Context, string, *RedisConfig) (*Response, error)
 	UpdateMySQLConfig(context.Context, string, *MySQLConfig) (*Response, error)
+	ListDBOptions(todo context.Context) (*DatabaseOptions, *Response, error)
```

Comment:
Maybe just `ListOptions`? It's a bit less repetitive, i.e. `client.Databases.ListDBOptions` vs  `client.Databases.ListOptions`

Reply:
good suggestion, will fix